pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -186,8 +186,10 @@ func makeDeviceName(bus string, devicePerBus map[string]int) string {
 	switch bus {
 	case "virtio":
 		prefix = "vd"
-	case "sata", "scsi":
+	case "sata":
 		prefix = "sd"
+	case "scsi":
+		prefix = "scd"
```

Comment:
I think `scd` prefix was a scsi cdrom, not a disk, iirc. I think it's has been long deprecated.

Reply:
Actually, I'm wrong, according to this : https://www.kernel.org/doc/Documentation/admin-guide/devices.txt

`/dev/scd` is replacing `/dev/sr`

Are you trying to add a CD-ROM?

Reply:
Hi, no . Actually the problem is that you can't mount a scsi disk and a sata disk at the same time.

Example scenario: I want to install a Windows VM on a scsi disk that needs virtio drivers. i need to mount the virtio-win in sata mode to be detected. Because scsi & sata have the same prefix but different counting mode, we have conflicts.

Reply:
I've changed the way i'm doing it by leaving the old system as-is but increment both sata & scsi counter instead.